### PR TITLE
Updated issue templates

### PR DESCRIPTION
**Pull request by patschi**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 1]
```diff
@@ -1,14 +1,29 @@
 ---
-name: Feature request
-about: Suggest an idea for this project
-
+name: 💡 Feature Request
+about: Suggest an idea for mailcow.
+labels: bug, Needs triage
```
Comment (ntimo) [inline .github/ISSUE_TEMPLATE/Feature_request.md]: A feature request should not have the label `bug`, I would suggest using `enhancement ` or `feature` or something like this.

**Comment by ntimo** [comment]
I really like the changes to the templates, and think they will be quite helpful 👍

File: .github/ISSUE_TEMPLATE/Feature_request.md (hunk 1 above)
Comment (patschi) [reply .github/ISSUE_TEMPLATE/Feature_request.md]: Oh, yeah, for sure. Let's say this was intended and was a test to see if someone is really checking the PR. Solved - thanks!

**Comment by andryyy** [comment]
Nice, thank you, Patrik!

I also appreciate the Extrawurst tag.

**Pull request merged**